### DISCUSSION TITLE
Change to Blazor Server?

### DISCUSSION THREAD

**Discussion by copperx** [fullstackhero/blazor-starter-kit#335] [Q&A]
Hi, would it be possible to configure this app as a Blazor Server app? Or would it need to be rewritten?

**Comment by chhinsras** [comment]
@copperx It is very easy to switch to Blazor Server. I personally use Blazor Server for development, while switching to WASM for production.

**Reply by oakuraape** [reply]
So what are the steps to change to Blazor Server?

**Reply by copperx** [reply]
It would be great to know how. I haven't dabbled in Blazor WASM, and I'm not sure about the steps. Anyone?

**Reply by copperx** [reply]
> So what are the steps to change to Blazor Server?

This might help.

https://www.youtube.com/watch?v=SkYQDPXw__c

**Reply by EdgeTheVeg** [reply]
If one switches to Blazor server, why not just keep it there?

Personally, I'm not seeing much benefit from the WASM mode in my environment and I really hate the slow load time.

**Comment by neozhu** [comment]
> Hi, would it be possible to configure this app as a Blazor Server app? Or would it need to be rewritten?
I have created it. refer to
https://github.com/neozhu/CleanArchitectureWithBlazorServer
